assets/scss/modules/_tooltip.scss

```diff
@@ -6,21 +6,17 @@
 /// Classes for the tooltip component
 
 .dp-library {
-  .icon-tip-icon {
```

Comment:
This I believe is from another PR right @GusBaamonde ?

Reply:
:open_mouth: